components/identity-core/org.wso2.carbon.identity.core/src/main/java/org/wso2/carbon/identity/core/util/IdentityUtil.java

```diff
@@ -214,11 +216,50 @@ public static IdentityCookieConfig getIdentityCookieConfig(String cookieName) {
         return identityCookiesConfigurationHolder;
     }
 
+    /**
+     * This method can use to check whether the legacy feature for the given legacy feature id is enabled or not
+     *
+     * @param legacyFeatureId      Legacy feature id.
+     * @param legacyFeatureVersion Legacy feature version.
+     * @return Whether the legacy feature is enabled or not.
+     */
+    public static boolean isLegacyFeatureEnabled(String legacyFeatureId, String legacyFeatureVersion) {
+
+        String legacyFeatureConfig;
+        if (StringUtils.isBlank(legacyFeatureId)) {
+            return false;
+        }
+        if (StringUtils.isBlank(legacyFeatureVersion)) {
+            legacyFeatureConfig = legacyFeatureId.trim();
+        } else {
+            legacyFeatureConfig = legacyFeatureId.trim() + legacyFeatureVersion.trim();
+        }
+        if (StringUtils.isNotBlank(legacyFeatureConfig)) {
+            LegacyFeatureConfig legacyFeatureConfiguration =
+                    legacyFeatureConfigurationHolder.get(legacyFeatureConfig);
+            if (legacyFeatureConfiguration != null && legacyFeatureConfiguration.isEnabled()) {
+                if (log.isDebugEnabled()) {
+                    log.debug("Legacy feature id: " + legacyFeatureConfiguration.getId() +
+                            " legacy feature version : " + legacyFeatureConfiguration.getVersion() +
+                            " is enabled: " + legacyFeatureConfiguration.isEnabled());
+                }
+                return legacyFeatureConfiguration.isEnabled();
+            }
+
+            if (log.isDebugEnabled()) {
+                log.debug("legacy feature is not configured or the configured legacy feature is empty. " +
+                        "Hence returning false ");
```

Comment:
Shall we change it as: 
log.debug("Legacy feature is not configured or the configured legacy feature is empty. " +
                        "Hence returning false.");

Reply:
Fixed https://github.com/wso2/carbon-identity-framework/pull/3112/commits/62f2f6eccc14c234cfb464ce3f267001f7fbdbee